src/reflow/mod.rs

```diff
@@ -117,7 +148,9 @@ fn reflow<'s>(origin: ContentOrigin, chunk: &'s CheckableChunk) -> Vec<Suggestio
                     }
                     Tag::Paragraph => {
                         // regular end of paragraph
-                        paragraph = store(cover.end, unbreakables.as_slice());
+                        // @todo the cover.end is exclusive and find_span() of CheckableChunk is inclusive, such that find_span()
```

Comment:
If this is the case, that is a bug. A `Range` by definition should never be inclusive.

Reply:
I changed the todo fmt from `@todo` to `TODO`.

Reply:
I thought that it's a bug when I used `find_span()` before I introduced `find_spans_inclusive()`, so there is no bug. Removed it.